sequila/sequila-core/benches/databio_benchmark.rs

```diff
@@ -0,0 +1,299 @@
+    sequila_config.interval_join_algorithm = algorithm.unwrap_or_default();
+
+    let config = SessionConfig::from(options)
+        .with_option_extension(sequila_config)
+        .with_information_schema(true)
+        .with_batch_size(2000)
+        .with_target_partitions(1);
+
+    SessionContext::new_with_sequila(config)
+}
+
+pub fn databio_benchmark(c: &mut Criterion) {
+    let root_path = env::var("BENCH_DATA_ROOT").unwrap_or("/data/bench_data/databio/".to_string());
+    let tables = [
+        Table {
+            id: 0,
+            name: String::from("chainRn4"),
+            non_flat: 6,
+            table_size: TableSize::Medium,
+        },
+        Table {
+            id: 1,
+            name: String::from("fBrain-DS14718"),
+            non_flat: 1,
+            table_size: TableSize::Small,
+        },
+        Table {
+            id: 2,
+            name: String::from("exons"),
+            non_flat: 2,
+            table_size: TableSize::Small,
+        },
+        Table {
+            id: 3,
+            name: String::from("chainOrnAna1"),
+            non_flat: 6,
+            table_size: TableSize::Medium,
+        },
+        Table {
+            id: 4,
+            name: String::from("chainVicPac2"),
+            non_flat: 8,
+            table_size: TableSize::Medium,
+        },
+        Table {
+            id: 5,
+            name: String::from("chainXenTro3Link"),
+            non_flat: 7,
+            table_size: TableSize::Large,
+        },
+        Table {
+            id: 6,
+            name: String::from("chainMonDom5Link"),
+            non_flat: 7,
+            table_size: TableSize::Large,
+        },
+        Table {
+            id: 7,
+            name: String::from("ex-anno"),
+            non_flat: 2,
+            table_size: TableSize::Small,
+        },
+        Table {
+            id: 8,
+            name: String::from("ex-rna"),
+            non_flat: 7,
+            table_size: TableSize::Medium,
+        },
+    ];
+
+    const QUERY: &str = r#"
+            SELECT
+                count(*)
+            FROM
+                s1 a, s2 b
+            WHERE
+                a.contig=b.contig
+            AND
+                a.pos_end>=b.pos_start
+            AND
+                a.pos_start<=b.pos_end
+        "#;
+
+    // Actutal test cases
+    let test_cases = vec![
+        TestCase {
+            s1: tables[0].clone(),
+            s2: tables[1].clone(),
+        },
+        TestCase {
+            s1: tables[0].clone(),
+            s2: tables[3].clone(),
+        },
+        TestCase {
+            s1: tables[0].clone(),
+            s2: tables[7].clone(),
+        },
+        TestCase {
+            s1: tables[0].clone(),
+            s2: tables[8].clone(),
+        },
+        TestCase {
+            s1: tables[7].clone(),
+            s2: tables[8].clone(),
+        },
+    ];
+    let algorithms = [
+        Some(Algorithm::Coitrees),
```

Comment:
why is it an Option? I think it's never `None`

Reply:
Fixed.